kratos/tests/test_kratos_parameters.py

```diff
@@ -11,102 +11,102 @@
 # input string with ugly formatting
 json_string = """
 {
-   "int_value" : 10,   "double_value": 2.0,   "bool_value" : true,   "string_value" : "hello",
+   "bool_value" : true, "double_value": 2.0, "int_value" : 10,
    "level1":
    {
      "list_value":[ 3, "hi", false],
      "tmp" : 5.0
-   }
+   },
+   "string_value" : "hello"

```

Comment:
Because internally the json reorder the values by types. This is just a property of the library. The important thing is that in the rests of things works exactly the same as the current parameters. But slightly faster, and much modern json library that rapidjson

Reply:
the point is the pretty pretting. internal storage is not guaranteed and rapidjson and nlohmann store it internally in different order